sdk/tables/azure-data-tables/migration_guide.md

```diff
@@ -0,0 +1,289 @@
+
+In the interest of simplicity, there are only two clients, `TableServiceClient` for account-level interactions and `TableClient` for table-level interactions. This is in contrast to the single `CosmosClient` used for interacting with the account level operations and the proxies used for interactions with specific resources (ie. `DatabaseProxy` and `ContainerProxy`).
+
+### Authenticating Clients
+
+In `azure-cosmosdb-table`:
+```python
+import os
+from azure.cosmosdb.table import TableService
+
+client = TableService(account_name=os.environ["ACCOUNT_URI"], account_key=os.environ["ACCOUNT_KEY"])
+```
+
+In `azure-data-tables`:
+```python
+import os
+from azure.core.credentials import AzureNamedKeyCredential
+from azure.data.tables import TableClient, TableServiceClient
+
+key = os.environ["TABLES_ACCOUNT_KEY"]
+name = os.environ["TABLES_ACCOUNT_NAME"]
+url = os.environ["TABLES_ACCOUNT_URL"]
+
+credential = AzureNamedKeyCredential(key=key, name=name)
+
+table_client = TableClient(account_url, "tablename", credential=credential)
+
+table_service_client = TableServiceClient(account_url, credential=credential)
+```
+
+
+### Table Level Scenarios
+
+#### Create and Delete
+
+In `azure-cosmosdb-table`:
+```python
+from azure.cosmosdb.table import TableService
+client = TableService(...)
+table_name = "tableName"
+created = client.create_table(table_name)
+# create_table returns True if a new table was created, False if the table already exists
+
+# Delete
+table_name = "deleteTableName"
+client.delete_table(table_name)
+# delete_table returns True if a table was deleted, False if the table does not exist
+```
+
+In `azure-data-tables`:
+```python
+from azure.data.tables import TableServiceClient, TableClient
+
+# You can create a table from the TableServiceClient or TableClient
+service_client = TableServiceClient.from_connection_string(conn_str)
+table_client = service_client.create_table("tableName")
+service_client.delete_table("tableName")
+
+# The create_table_if_not_exists
+table_client = TableClient.from_connection_string(conn_str, table_name="tableName")
+table_client.create_table_if_not_exists()
+table_client.delete_table()
+```
+
+#### List and Query
+
+In `azure-cosmosdb-table`:
+```python
+from azure.cosmosdb.table import TableService
+client = TableService(...)
+
+# In the azure-cosmosdb-table library, there is no query table method
+tables = list(service.list_tables())
+for table in tables:
+    print(table.name)
+```
+
+In `azure-data-tables`:
+```python
+from azure.data.tables import TableServiceClient
+
+service_client = TableServiceClient.from_connection_string(conn_str)
+
+for table in service_client.list_tables():
+    print(table.name)
+
+query_filter = "TableName eq @name1 or TableName gt @name2'
+parameters = {
+    "name1": "myTableName1",
+    "name2": "myTableName2",
+}
+
+for table in service_client.query_tables(query_filter, parameters=parameters):
+    print(table.name)
+```
+
+### Entity Level Scenarios
```

Comment:
Same thing here. Why should customers use Tables for each scenario? Better? Faster? Stronger?